docs/kb/developer-resources/running-commands-on-the-host-os.md

```diff
@@ -49,14 +49,14 @@ components:
           - host_path: /keys
             container_path: /keys
             is_ephemeral: true
-            
+
         cmd: '["/create_ssh_key.sh"]'
         config_files:
           - filename: /create_ssh_key.sh
             file_mode: 0700
             contents: |
               #!/bin/bash
-              
+
```

Comment:
Does this break it? I would think Yaml parser would continue as long as you stayed at that indentation level.